things/postgres/channels.go

```diff
@@ -333,7 +332,7 @@ func (cr channelRepository) hasThing(chanID, thingID string) error {
 // dbMetadata type for handling metadata properly in database/sql
 type dbMetadata map[string]interface{}
 
-// Scan - Implement the database/sql scanner interface
+// Scan - Implement the database/sql scanner interface.
```

Comment:
No need for a full-stop at the end of the comment.

Reply:
I recommend that we follow go standard. Don't write `-` after method name; just write a full sentence that begins with the method name. For example:
```
// Scan implements the database/sql scanner interface.
```